yoti-sdk-api/src/main/java/com/yoti/api/client/ActivityDetails.java

```diff
@@ -1,7 +1,6 @@
 package com.yoti.api.client;
 
-import static com.yoti.api.client.spi.remote.util.Validation.notNull;
-
+import static com.yoti.validation.Validation.notNull;
```

Comment:
is this right? the diff doesn't show any _new_ `notNull` calls so I don't see why suddenly this import is needed...

Reply:
ah ok - ideally if that import change had been done in it's own commit it would have been clearer 😉